library/swift/src/GRPCStreamEmitter.swift

```diff
@@ -43,7 +43,9 @@ public final class GRPCStreamEmitter: NSObject {
   }
 
   /// Close this connection.
-  public func close() {
-    self.underlyingEmitter.close()
+  public func close(_ data: Data) {
```

Comment:
Why not leave the public interface unchanged and simply close with `Data()` under the hood? I think this is a bit confusing since it's not immediately clear what should be sent here.